js/modulestest/runtime.go

```diff
@@ -107,3 +112,45 @@ func (r *Runtime) innerSetupModuleSystem() error {
 	impl := modules.NewLegacyRequireImpl(r.VU, ms, url.URL{})
 	return r.VU.RuntimeField.Set("require", impl.Require)
 }
+
+//go:embed wptutils/*
+var wptutils embed.FS
+
+// NewRuntimeForWPT will create a new test runtime like NewRuntime, but ready to be used
+// for Web Platform Tests (https://github.com/web-platform-tests/wpt).
+func NewRuntimeForWPT(t testing.TB) *Runtime {
+	var err error
+	runtime := NewRuntime(t)
+
+	// We want to make the [console.log()] available for Web Platform Tests, as it
+	// is very useful for debugging, because we don't have a real debugger for JS code.
+	logger := runtime.VU.InitEnvField.Logger
+	require.NoError(t, runtime.VU.RuntimeField.Set("console", newConsole(logger)))
+
+	// We also want to make [timers.Timers] available for Web Platform Tests.
+	for k, v := range timers.New().NewModuleInstance(runtime.VU).Exports().Named {
+		require.NoError(t, runtime.VU.RuntimeField.Set(k, v))
+	}
+
+	// We compile the Web Platform Tests harness scripts into a goja.Program,
+	// and execute them in the goja runtime in order to make the Web Platform
+	// assertion functions available to the tests.
+	files, err := fs.ReadDir(wptutils, "wptutils")
+	require.NoError(t, err)
+
+	for _, file := range files {
+		// Skip directories for safety,
+		// as we expect all files to be present in the root.
+		if file.IsDir() {
+			continue
+		}
+
+		program, err := CompileFileFromFS(wptutils, fmt.Sprintf("wptutils/%s", file.Name()))
+		require.NoError(t, err)
+
+		_, err = runtime.VU.Runtime().RunProgram(program)
+		require.NoError(t, err)
+	}
+
+	return runtime
+}
```

Comment:
I would really prefer if this isn't part of this module but part of the one that is being tested. This goes for practically all teh changes in `modulestest` although arguably the `console` one might be a good candidate to be extracted in a separate module and used both in the runner and `modulestest`

Reply:
I can do that if you agree that's a good idea. I'd say I'm even surprised by to see that we have no other needs to expose `console`.

Reply:
For the rest, I'm happy to move it as well, although that's a function that arguably could be used by other modules relying on WPTs. But lets copy & paste for now,